Dockerfile

```diff
@@ -177,7 +178,7 @@ RUN set -ex && \
 HEALTHCHECK --start-period=20s \
     CMD curl -f http://localhost:8228/health || exit 1
 
-USER 1000
-
+USER anchore:anchore
```

Comment:
This should not be changed from the value in master, the value is required for dynamic UID mapping in OpenShift

Reply:
@zhill Good catch, I've reverted this change. I'm assuming this relates to this functionality -> https://www.openshift.com/blog/a-guide-to-openshift-and-uids . For future reference, where can I find out more about how Anchore interacts with OCP's UID management?